### DISCUSSION TITLE
"error": "--from is not a valid sha pointer: 'HEAD~20'

### DISCUSSION THREAD

**Discussion by cristiaan30033** [scolladon/sfdx-git-delta#950] [Q&A]
Hi community,
how can I fix this error? obviously, I can't decrease the depth, I need 20 or more previously. it is something I can configure in somewhere?

sfdx sgd:source:delta --to HEAD --from HEAD~20 --output .

**"error": "--from is not a valid sha pointer: 'HEAD~20' (If in CI/CD context, check the fetch depth is properly set)",**

**Comment by scolladon** [comment]
HI @cristiaan30033 

Is it a CI usage ?
If yes then what is your yml configuration file ? I'm interested in the code checkout section to make sure the fetch depth is set.
